backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -1,22 +1,22 @@
 package gov.cdc.usds.simplereport.db.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PersonName;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PersonRole;
```

Comment:
I love that IntelliJ and Eclipse have different default orders for organizing exports. Get it together, people! 🤣